### ISSUE TITLE
API Extension for Leader Experience

### ISSUE THREAD

**Issue by jterps08** [PowerlineApp/powerline-server#22]
We need the ability for an authorized group owner or manager to create group, create leader content (e.g. polls, fundraisers, etc.), and manage all group functionality from the unified web app instead of the current admin control panel. This requires extending the API endpoints for access to functionality currently contained in the FrontEndBundle (admin.powerli.ne) including group registration, group subscriptions, group management settings, and leader content creation. Once this is complete, the attached pages should all be possible.

Superuser from admin.powerli.ne is out of scope for this effort. Elected Representatives should be considered in the planning and approach. Ideally, elected representatives are addressed simultaneously to group managers with this exercise. I will create a separate post about how elected representatives work currently for consideration.

Related: Group registration via web pages like these mockups should be possible https://drive.google.com/open?id=0B2Loy21ehREyUlpPNWZsNFBadkE
Related: Creating content for a group via web pages like these mockups should be possible https://drive.google.com/open?id=0B2Loy21ehREyUmpqd0V2cGh5MWs
Related: Controlling group settings (based on subscription level) via web pages like these mockups should be possible https://drive.google.com/open?id=0B2Loy21ehREyUmpqd0V2cGh5MWs

![new in red create content component for web app extensions](https://cloud.githubusercontent.com/assets/7582457/11853280/2a33eda0-a40b-11e5-82dd-1fde96695bec.png)
![new assigning group manager within control component for web app extensions](https://cloud.githubusercontent.com/assets/7582457/11853281/2a373866-a40b-11e5-816b-691d42274edc.jpg)
![new control component for web app extensions](https://cloud.githubusercontent.com/assets/7582457/11853279/2a3387b6-a40b-11e5-8f65-71004c57f67c.jpg)
